examples/stream.ru

```diff
@@ -8,17 +8,20 @@
 #   puma stream.ru                # gem install puma
 
 require 'sinatra/base'
+require 'sinatra/stream/reel'
 
 class Stream < Sinatra::Base
   get '/' do
-    content_type :txt
+    #content_type :txt
 
-    stream do |out|
-      out << "It's gonna be legen -\n"
+    #stream do |out|
+    Rack::StreamingResponse.new do |out|
+      out.hijack!(env)
```

Comment:
@zzak do you have some insight about accessing the request env from within the response, so that I don't have to explicitly hijack within the response block? I really don't like that, by the the time "out" is exposed in the block, the socket isn't hijacked yet. My rack-fu is not that strong.